src/docs/asciidoc/en/rlsnotes/rlsnotes40/_api-and-ods-40.adoc

```diff
@@ -436,7 +436,7 @@ batch->registerBlob(&status, &realId, &msg->desc);
 ----
 
 If the BLOB policy is making the Firebird engine generate BLOB IDs then this code is enough to correctly register an existing BLOB in a batch.
-In other cases you will have to assign to `msg>>desc` the ID that is correct from the point of view of the batch.
+In other cases you will have to assign to `msg->desc` the ID that is correct from the point of view of the batch.
```

Comment:
This will trigger a character merge to →, so change it to `msg\->desc` or `++msg->desc++` (or `msg++->++desc`, but I think that is less readable)